src/components/project/project.rules.ts

```diff
@@ -405,15 +407,19 @@ export class ProjectRules {
               type: TransitionType.Approve,
               label: 'Confirm Project 🎉',
               notifiers: async () => [
-                ...(await this.getRoleEmails(Role.Controller)),
+                ...(await this.financialApproverRepo.read([projectType])).map(
+                  (fa) => fa.user.email!,
+                ),
```

Comment:
We should use the `user.id` here instead. Otherwise these emails are treated as distribution lists and could be filtered out erroneously.
https://github.com/SeedCompany/cord-api-v3/blob/5e49de34c52a891c5d39ccce2bac9496ad38e756/src/components/project/project.rules.ts#L1012-L1016
Also these would be faked to the root user id instead of using the actual user.
https://github.com/SeedCompany/cord-api-v3/blob/5e49de34c52a891c5d39ccce2bac9496ad38e756/src/components/project/project.rules.ts#L1114-L1133

----

Also you could follow suit of this
https://github.com/SeedCompany/cord-api-v3/blob/5e49de34c52a891c5d39ccce2bac9496ad38e756/src/components/project/project.rules.ts#L86-L87
And provide a helper function to call in each use rather than duplicating all of it.
```ts
const financialApprovers = () => this.financialApproverRepo
  .read(projectType)
  .map(({ user }) => user.id);

Reply:
I guess you could check for email and filter those out without. But that wouldn't save anything in practice probably. These users wouldn't be approvers and not have email addresses. Also we still filter them out later if that did happen